src/auth/registration/AddressField.tsx

```diff
@@ -44,7 +48,7 @@ export function AddressField() {
         useTopLabel
       />
 
-      <Stack direction="row" spacing={1}>
+      <Stack direction={isMobile ? "column" : "row"} spacing={isMobile ? 2 : 1}>
```

Comment:
Is it possible to use css application of breakpoints, rather than js based as is working here?

So this should work identically, but is implemented in css rather than requiring a re-render.

(Most MUI css props support prefixing with the breakpoint name which it then applies the listed style with a media query, e.g. in this case `@media (min-width: 900px)`)

I don't think you can quite do this in every case - it doesn't work for non-css props like `variant` but this style is much preferred if possible.

```suggestion
     <Stack direction={{ xs: "column", md: "row" }} spacing={{ xs: 2, md: 1 }}>
```